src/OrchardCore.Modules/OrchardCore.OpenId/Services/OpenIdValidationService.cs

```diff
@@ -110,15 +110,60 @@ public async Task<ImmutableArray<ValidationResult>> ValidateSettingsAsync(OpenId
                     }));
                 }
 
+                if (settings.Authority.Scheme != Uri.UriSchemeHttps)
```

Comment:
Please update this line to allow HTTP addresses and use a case-insensitive comparison.